packages/bpk-docs/src/layouts/NeoSideNavLayout/SectionsList.scss

```diff
@@ -47,6 +48,7 @@ $bpk-heading-button-padding: $bpk-spacing-md;
   }
 
   &__heading-button-arrow {
+    margin-left: $bpk-spacing-sm;
```

Comment:
Needs RTL support

Reply:
#660 has been merged now you can use the new mixins